test/Checkout/Tests/Payments/RequestApmPaymentsIntegrationTest.php

```diff
@@ -43,11 +42,11 @@ public function shouldMakeBalotoPayment(): void
 
         $paymentRequest = new PaymentRequest();
         $paymentRequest->source = $requestSource;
-        $paymentRequest->amount = 10000;
+        $paymentRequest->amount = 100000;
```

Comment:
Can you please reduce the amount to 10 or something like this? It's not safe to test with big amounts, imagine someone decides to run these tests with production credentials :)))

Reply:
I don't understand why is this important on this SDK and not in others besides the failure is regarding of the amount, using other amount rather than 100000 throws 
`422 Unprocessable Entity` response: {"request_id":"09f3607a-3500-41e1-ab84-ab45439b6651","error_type":"request_invalid","error_codes":["amount_invalid"]}  [] []`